Python_Engine/Compute/DownloadPython.cs

```diff
@@ -33,22 +37,24 @@ public static partial class Compute
         /**** Public Methods                            ****/
         /***************************************************/
 
-        public static void CopyEmbeddedResourceToFile(string resourceName, string targetPath, bool force = false)
+        public static bool DownloadPython()
         {
-            if (force || !File.Exists(targetPath)) 
-            {
-                Assembly assembly = typeof(Compute).Assembly;
-                var key = assembly.GetManifestResourceNames().FirstOrDefault(x => x.Contains(resourceName));
-                using (Stream stream = assembly.GetManifestResourceStream(key))
-                using (var file = new FileStream(targetPath, FileMode.Create))
-                {
-                    if (stream == null)
-                        throw new ArgumentException($"Resource name '{resourceName}' not found!");
-                    stream.CopyTo(file);
-                }
-            }
+            string home = Query.EmbeddedPythonHome();
+            if (!Directory.Exists(home))
+                Directory.CreateDirectory(home);
+
+            string destination = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData), "BHoM", $"{EMBEDDED_PYTHON}.zip");
+
+            RunCommand($"curl {EMBEDDED_PYTHON_URL} -o {destination}");
+            return true;
```

Comment:
If someone already installed Python Toolkit, so the folder already exists, will this method overwrite it without throwing an error to the user about the file already existing? (I guess I could test, but I haven't started that part of the review yet!)

Reply:
Nope, this is downloading the archive, not the folder.
The archive is inflated by the `InstallPython` method, the check happens there